pkg/querier/blocks_store_queryable_test.go

```diff
@@ -863,7 +864,7 @@ func TestBlocksStoreQuerier_Select(t *testing.T) {
 				var actualValues []valueResult
 
 				it := set.At().Iterator()
-				for it.Next() {
+				for it.Next() == chunkenc.ValFloat {
```

Comment:
`!= none` and the check the actual type to ensure it's always a float.

Reply:
done